### PR TITLE
Updating to v2.0.0beta2

### PR DESCRIPTION
**Pull request by erikvansebille**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by willirath** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ requirements:
     - clang_osx-64  # [osx]
     - m2w64-toolchain  # [win]
     - matplotlib >=2.0.2
-    - netcdf4 >=1.1.9
+    - netcdf4 >=1.1.9,<=1.4.1
     - numpy >=1.9.1
```
Comment (willirath) [inline recipe/meta.yaml]: We should follow https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L462 and pin numpy.

Comment (erikvansebille) [reply recipe/meta.yaml]: Sure, happy to pin numpy. Is that simply a matter of changing 
```
    - numpy >=1.9.1
```
to
```
    numpy:
      - 1.9    # [linux64 or osx]
      - 1.11   # [aarch64 or ppc64le]
      - 1.11   # [win]
```
?

**Comment by willirath** [comment]
> * [ ]  pin numpy differently for linux and windows.
> * [ ]  Test example-data download (See https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L462)

Link was meant to point here: https://github.com/OceanParcels/parcels/issues/533

As @delandmeterp saw failing `parcels_get_examples` on Windows, we should add a testfor this command-line tool: http://conda-forge.org/docs/maintainer/adding_pkgs.html#command-line-utilities

File: recipe/meta.yaml (hunk 1 above)
Comment (willirath) [reply recipe/meta.yaml]: I think it's
```yaml
    - numpy >=1.9  # [linux64 or osx]
    - numpy >=1.11  # [win]
```

Not sure if we need the others.

**Comment by willirath** [comment]
Windows command-line tool is failing.  So I'd propose adding `  # [linux or osx]`  (Note the two spaces!) behind the command and investigate this further in a different issue.

**Pull request merged**